### PR TITLE
PlotOutline to DialogPVRChannelsOSD

### PR DESCRIPTION
**Pull request by walterheisenberg**
added PlotOutline and changed progressbar a little

### CODE REVIEW

File: 16x9/DialogPVRChannelsOSD.xml [hunk 1]
```diff
@@ -217,22 +217,68 @@
...
+							<control type="textbox">
+								<width>500</width>
+								<height>413</height>
+								<font>font14_textbox</font>
+								<textcolor>grey</textcolor>
+								<label>$INFO[Container(11).ListItem.Plot]</label>
+								<autoscroll delay="8000" time="2000" repeat="10000">true</autoscroll>
+								<visible>Control.IsVisible(5555)</visible>
+							</control>
+							<control type="image" id="5555">
```
Comment (mikesilvo164) [inline 16x9/DialogPVRChannelsOSD.xml]: I don't understand why you are basing visibles on this image...  :confused:

Comment (walterheisenberg) [inline 16x9/DialogPVRChannelsOSD.xml]: I thought the same, but it was in the code a long time, so I left this in :~

**Comment by mikesilvo164** [comment]
Really need a Pic for this... Hard to test PR stuff for PVR ATM.

**Comment by walterheisenberg** [comment]
Here are "before" and "after"-screenshots. I marked the changes on the second picture.
http://imgur.com/a/4h3rE

**Comment by mikesilvo164** [comment]
Time to delete these as the whole Plot Outline thing never made sense to me anyway... Sorry.

**Comment by walterheisenberg** [comment]
PlotOutline is not used for LiveTV anymore.
You should add "Episode- and SeasonNr:" and "Episodename" on the same place of this PR

**Comment by mikesilvo164** [comment]
So like this?
img

**Pull request closed without merging**